plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -698,22 +707,15 @@ fu_cros_ec_usb_device_jump_to_rw(FuCrosEcUsbDevice *self)
 						   &response,
 						   &response_size,
 						   FALSE,
-						   NULL)) {
+						   &error_local)) {
 		/* bail out early here if subcommand failed, which is normal */
+		g_debug("ignoring failure: jump to rw: %s", error_local->message);
 		return TRUE;
 	}
 
 	/* Jump to rw may not work, so if we've reached here, initiate a
 	 * full reset using immediate reset */
-	subcommand = FU_CROS_EC_UPDATE_EXTRA_CMD_IMMEDIATE_RESET;
-	fu_cros_ec_usb_device_send_subcommand(self,
-					      subcommand,
-					      command_body,
-					      command_body_size,
-					      &response,
-					      &response_size,
-					      FALSE,
-					      NULL);
+	fu_cros_ec_usb_device_reset_to_ro(self);
```

Comment:
I'm confused by this -- it doesn't match the comment at all now. What's the aim?

Reply:
hm... any suggestion?
Jump to RW command is above.
fu_cros_ec_usb_device_reset_to_ro() is literally wrapper for reset (`FU_CROS_EC_UPDATE_EXTRA_CMD_IMMEDIATE_RESET`) command.
Still need the delay. Private flag wouldn't be used in workflow after *_attach() from there, this function is called

Reply:
tbh `FU_CROS_EC_USB_DEVICE_FLAG_REBOOTING_TO_RO` must be set as far as I understood if we reset the device to bootloader, but as I mentioned -- this flag is not used in a workflow after *_attach()